mindmeld/converter/generic_config.txt

```diff
@@ -1,31 +0,0 @@
-ENTITY_RECOGNIZER_CONFIG = {
```

Comment:
we should keep this since the rasa converter is also using this

Reply:
We should be using the default settings. The only difference between this and the default is stemming being enabled, which is not necessary for RASA or DF.